tests/org.jboss.tools.aerogear.ui.bot.test/src/org/jboss/tools/aerogear/ui/bot/test/utils/ResourceUtils.java

```diff
@@ -0,0 +1,24 @@
+package org.jboss.tools.aerogear.ui.bot.test.utils;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.nio.MappedByteBuffer;
+import java.nio.channels.FileChannel;
+import java.nio.charset.Charset;
+
```

Comment:
I did not find such a functionality neither in RedDeer or JBT so probably it's best to move it to org.jboss.reddeer.core.util and I prefer to not name it ResourceUtils because it's evoking utils for Eclipse resources but this I will leave to you